spec/ADAS/ADAS.vspec

```diff
@@ -131,6 +150,28 @@ ObstacleDetection.IsError:
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
+ObstacleDetection.Distance:
+  datatype: float
+  type: sensor
+  unit: m
+  description: Distance in meters to detected object
```

Comment:
Can never be negative, or?

And distance is distance from the specified "zone". I.e. an object just in front of the front number plate may be reported with distance==0.01 meter by Front.Center, but with distance==0.80 meter by Front.Right?

Reply:
Distance from sensor and would be in zonal instance eg Front Right.

Wording this detail tricky, let's hash out in comments, slack, call or other. 

Description: Distance in meters from detected object to perimeter, physical boundary of vehicle at the identified zone (eg Front Right).. Cannot be a negative number, expected value is  positive float (unsigned).